packages/editor/src/components/simple-editing-mode-selector/index.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	Button,
+	SVG,
+	Path,
+	privateApis as componentsPrivateApis,
+} from '@wordpress/components';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { forwardRef } from '@wordpress/element';
+import { edit } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../lock-unlock';
+import { store as editorStore } from '../../store';
+import { TEMPLATE_POST_TYPE } from '../../store/constants';
+
+const { DropdownMenuV2 } = unlock( componentsPrivateApis );
+
+const selectIcon = (
+	<SVG
+		xmlns="http://www.w3.org/2000/svg"
+		width="24"
+		height="24"
+		viewBox="0 0 24 24"
+	>
+		<Path d="M9.4 20.5L5.2 3.8l14.6 9-2 .3c-.2 0-.4.1-.7.1-.9.2-1.6.3-2.2.5-.8.3-1.4.5-1.8.8-.4.3-.8.8-1.3 1.5-.4.5-.8 1.2-1.2 2l-.3.6-.9 1.9zM7.6 7.1l2.4 9.3c.2-.4.5-.8.7-1.1.6-.8 1.1-1.4 1.6-1.8.5-.4 1.3-.8 2.2-1.1l1.2-.3-8.1-5z" />
+	</SVG>
+);
+
+function SimpleEditingModeSelector( props, ref ) {
+	const { postType, renderingMode } = useSelect(
+		( select ) => ( {
+			postType: select( editorStore ).getCurrentPostType(),
+			renderingMode: select( editorStore ).getRenderingMode(),
+		} ),
+		[]
+	);
+
+	const { setRenderingMode } = useDispatch( editorStore );
```

Comment:
I don't understand why we're using the "rendering mode" instead of the editor mode (navigation mode) for this. These two feel to me like different things and that we shouldn't be touching the rendering modes for this PR.

Reply:
I expect it was a quick way to get up and running. There a couple of things that need to be solved with using editor mode:
- Editor mode already uses a value of `edit` to represent what we want to call 'design'. Not unsolvable, but definitely awkward. We might want to deprecate the current values and go for something like `design` and `content` as the values.
- From a user perspective, the editing mode seems like it can work alongside `zoom-out` mode, but `zoom-out` is already an editor mode so with the way it works right now the two couldn't be active at the same time. Seems like we'd need to figure out how we want the two to work together and exactly what the right primitives are for representing these modes.

Reply:
I used rendering mode because it's in `editorStore` not `blockEditorStore` and I was thinking that this feature isn't really something that all block editors will want.

Reply:
> From a user perspective, the editing mode seems like it can work alongside zoom-out mode

I don't think we should allow this. When zoomed out, the text is small and not conducive to editing.

To me there's three "modes":
- Zoomed out. Focus is on patterns and previewing the whole page
- Edit mode. Focus is on patterns and editing the content
- Design mode. Focus is on blocks

So yeah sounds like editor mode is a better fit 👍